tools/rostest/package.xml

```diff
@@ -21,7 +21,7 @@
   <run_depend>rosmaster</run_depend>
   <run_depend>rospy</run_depend>
   <run_depend>rosunit</run_depend>
-
+  <run_depend>rospy_message_converter</run_depend>
```

Comment:
I am not sure this new external dependency in an already release ROS distro is desired in a package that low in the stack (being part of the ROS core).

I don't know how easy it would be to remove the need for this dependency or if it would be easier to release these new features as a standalone package?

Reply:
@aremazeilles Any thought on this?

Reply:
Sorry for the delay in answering. Removing the dependency would require copying significant code from  `rospy_message_converter` into that package, which I am not sure it is the best strategy.
So maybe the second option you propose, making a standalone package, might be a better solution. What do you think?

Reply:
> making a standalone package, might be a better solution. What do you think?

Yes, I would recommend that.